docker/docker-compose.yml

```diff
@@ -9,4 +9,5 @@ services:
     environment:
       SIRIUS_URL: http://docker.for.mac.localhost:8080
       SIRIUS_PUBLIC_URL: http://localhost:8080
+      PRO_HUB_URL: http://localhost:8888/supervision/deputies/professional
```

Comment:
The pro hub is being removed in the near future so is it possible to use the combined hub instead? or is that hub missing some functionality that's required here?

Reply:
That is a good point, i'll check with Phil as I think the original ticket was for the firm hub

Reply:
Checked with Rich (as Phil is on AL) and he said because it's not very much to change it's fine as it is for now